Installation/Schema/metadata.tooi_adviezen.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+    "title": "Metadata TOOI adviezen",
+    "$id": "https://openwoo.app/schemas/metadata.tooi_adviezen.schema",
+    "$schema": "https://docs.openwoo.app/schemas/metadata.tooi_adviezen.schema",
```

Comment:
For Entities / Schemas we always need to use:
`"$schema": "https://docs.commongateway.nl/schemas/Entity.schema.json"`